olga/functional_tests/test_graph_metrics.py

```diff
@@ -0,0 +1,69 @@
+"""
+Test for graphs metrics.
+"""
+
+from django.test import TestCase
+
+from ddt import ddt, data, unpack
+
+from olga.functional_tests.utils import SetUp, html_target
+
+from olga.analytics.tests.factories import InstallationStatisticsFactory
+
+# pylint: disable=invalid-name, attribute-defined-outside-init
+
+
+factory_stats = InstallationStatisticsFactory()
+
+
+@ddt
+class TestGraphsMetricsWithoutStatistics(TestCase):
+    """
+    Test html render graphs metrics if statistics does not exist.
```

Comment:
> You testing data or rendering here? If data, why not test api itself? If rendering, what exactly you are testing?

I am testing, that my view pass to context true metrics data and user gets corresponding values.
![Global Metrics](https://user-images.githubusercontent.com/22666467/27918381-7b5efa54-6277-11e7-86a5-ede9d9a0302e.png)